### ISSUE TITLE
nested LDAP groups

### ISSUE THREAD

**Issue by bartjackbakker** [Ylianst/MeshCentral#4839]
We do use the ldap login options. Now im trying to configure the groups.
First of all is did create a couple of security groups in our AD and the sync is working find even the filter function is oke.

but now i want to add users and other groups. The users are visible in mesh and added to the group but users in a nested group
are not visible in mesh.

Is there a way to make this work?

As you can see in the image below the 2 users are visible but the werkvoorbereiders not.
![image](https://user-images.githubusercontent.com/9839745/206415779-b6ead0dd-9339-4c25-971e-d41db68a8a09.png)

**Comment by bartjackbakker** [comment]
![image](https://user-images.githubusercontent.com/9839745/206415779-b6ead0dd-9339-4c25-971e-d41db68a8a09.png)

**Comment by Pinkbyte** [comment]
+1 for support for nested groups!

**Comment by si458** [comment]
i dont believe its currently possible, but we could look into it, so i have added correct tag to remind us!

**Comment by kno005** [comment]
The work around we use is setting the search filter as:
"searchFilter": "(&(sAMAccountName={{username}})(memberOf:1.2.840.113556.1.4.1941:=CN=MeshCentralAccess,OU=Groups,DC=contoso,DC=com))"

The only downside is users not in the group or nested groups get a Login failed message instead of access denied.

**Comment by bartjackbakker** [comment]
@kno005 Not perfect but beter than nothing. Hopefully there wil be a proper implementation for the nested groups

**Comment by MarkStephan89** [comment]
Is there a chance it will still work?

**Comment by Pinkbyte** [comment]
@MarkStephan89, workaround works perfectly at least on 1.2.0
